### PR TITLE
Added 3D button

### PR DESCRIPTION
**Pull request by Charu271**
Issue:

### CODE REVIEW

File: src/components/animated/_key.less [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  box-shadow: inset 0px -12px rgb(44, 40, 40);
+  &:active {
+    top: 2px;
+    border-top: 2px solid #2b1800;
+    background-color: rgb(71, 69, 69);
+    box-shadow: inset 0px -8px 0px rgb(44, 40, 40);
+  }
+}
+.key-btn.btn-rounded {
+  border-radius: 30px;
```
Comment (shahednasser) [inline src/components/animated/_key.less:113]: change `btn-rounded` to `rounded-btn`

**Review by shahednasser — CHANGES_REQUESTED** [review]
The buttons look good but I have a few remarks:

1. It seems you have a lot of repeated code in `_key.less` if you can refactor it to make sure there isn't a lot of repeated code then that would be great. If you're not sure how to do it, I can edit the file once other requested changes are implemented.
2. Please use for colors their associated variables in `src/sbuttons.less` instead of writing them directly.
3. Did you compile the code or just copy it into `dist/sbuttons.css`? You need to import `_key.less` in `src/components/_animated.less` then run `npm run compile`. You can't manually edit the `dist/sbuttons.css` file.

**Comment by Charu271** [comment]
I will make all the required changes

**Comment by Charu271** [comment]
I have done all the changes you asked for but again facing issue while
making pull request as it is having conflicts

**Comment by shahednasser** [comment]
You can make a pull request even if there are conflicts. You can also try to update your repo to the changes made in this one.

**Comment by Charu271** [comment]
I have created the pull request. Kindly go through it

**Pull request closed without merging**